NavigationNotice/NavigationNotice.swift

```diff
@@ -376,8 +376,20 @@ open class NavigationNotice {
                     return nil
                 }
 
-                let superWindow = (superview as? UIWindow) ?? superview.window
-                return (superWindow != self) ? superWindow : nil
+                let targetView: UIView? = {
+                    if let superWindow = superview as? UIWindow {
+                        return superWindow
+
+                    // SeeAlso: https://github.com/abema/NavigationNotice/pull/10
+                    } else if let superview = superview.superview, NSStringFromClass(type(of: superview)) == "UIDropShadowView" {
+                        return superview.window
+
+                    } else {
+                        return superview
+                    }
+                }()
+
+                return (targetView != self) ? targetView : nil
```

Comment:
Given the original code handles only UIWindows, could recursively iterating superviews and use the window of the "root" view be enough here?
I'm wondering if using `"UIDropShadowView"`, even with a literal name, can be a potential risk as an usage of internal API 🤔

Reply:
If recursively captures UIWindow like old code, tap event of button doesn’t fired.
This reason why `HitWindow.hitTest` returns `UIWindow` not `UIButton` when tapped button on NavigationNotice.

So I handle `UIDropShadowView` as `UIWindow` by checking superview.